src/CabanaPD_Particles.hpp

```diff
@@ -462,6 +469,8 @@ class Particles<MemorySpace, PMB, Dimension>
 #ifdef Cabana_ENABLE_HDF5
     Cabana::Experimental::HDF5ParticleOutput::HDF5Config h5_config;
 #endif
+
+    Timer _timer;
```

Comment:
@streeve Why is this at the end of a function, just before "};"?

Reply:
End of the class definition. It's just a habit for protected variables honestly